src/groups/mqb/mqbblp/mqbblp_pushstream.t.cpp

```diff
@@ -0,0 +1,214 @@
+                                                      itGuid2,
+                                                      itApp1);
+
+    ps.add(element4);
+
+    mqbu::CapacityMeter dummyCapacityMeter("dummy", s_allocator_p);
+    bmqt::Uri           dummyUri("dummy", s_allocator_p);
+    mqbconfm::Domain    dummyDomain(s_allocator_p);
+
+    mqbs::InMemoryStorage dummyStorage(dummyUri,
+                                       mqbu::StorageKey::k_NULL_KEY,
+                                       mqbs::DataStore::k_INVALID_PARTITION_ID,
+                                       dummyDomain,
+                                       &dummyCapacityMeter,
+                                       s_allocator_p);
+
+    mqbconfm::Storage config;
+    mqbconfm::Limits  limits;
+
+    config.makeInMemory();
+
+    limits.messages() = bsl::numeric_limits<bsls::Types::Int64>::max();
+    limits.bytes()    = bsl::numeric_limits<bsls::Types::Int64>::max();
+
+    mwcu::MemOutStream errorDescription(s_allocator_p);
+    dummyStorage.configure(errorDescription,
+                           config,
+                           limits,
+                           bsl::numeric_limits<bsls::Types::Int64>::max(),
+                           0);
+
+    {
+        mqbblp::PushStreamIterator pit(&dummyStorage,
+                                       &ps,
+                                       ps.d_stream.begin());
+
+        ASSERT(!pit.atEnd());
+        ASSERT_EQ(pit.numApps(), 2);
+
+        ASSERT_EQ(element1, pit.element(0));
+        ASSERT_EQ(element3, pit.element(1));
+
+        ASSERT(pit.advance());
+        ASSERT_EQ(pit.numApps(), 2);
+
+        ASSERT_EQ(element2, pit.element(0));
+        ASSERT_EQ(element4, pit.element(1));
+
+        ASSERT(!pit.advance());
+    }
+
+    {
+        mqbblp::VirtualPushStreamIterator vit(subQueueId1,
+                                              &dummyStorage,
+                                              &ps,
+                                              ps.d_stream.begin());
+
+        ASSERT(!vit.atEnd());
+        ASSERT_EQ(vit.numApps(), 1);
+
+        ASSERT_EQ(element1, vit.element(0));
+
+        vit.advance();
+
+        ASSERT(!vit.atEnd());
+        ASSERT_EQ(vit.numApps(), 1);
+
+        ASSERT_EQ(element4, vit.element(0));
+
+        vit.advance();
+
+        ASSERT(vit.atEnd());
+    }
+
+    ps.remove(element2);
+    ps.destroy(element2, false);
+    ps.remove(element3);
+    ps.destroy(element3, false);
+}
+
+// ============================================================================
+//                                 MAIN PROGRAM
+// ----------------------------------------------------------------------------
+
+int main(int argc, char* argv[])
+{
+    TEST_PROLOG(mwctst::TestHelper::e_DEFAULT);
+
+    bmqt::UriParser::initialize(s_allocator_p);
+
+    switch (_testCase) {
+    case 0:
+    case 1: test1_basic(); break;
+    case 2: test2_iterations(); break;
```

Comment:
Typically we have a descending order, not sure if UTs runner relies on this
```suggestion
    case 2: test2_iterations(); break;
    case 1: test1_basic(); break;
```

Reply:
hmm.  I thought otherwise.  For example, `mqbblp_remotequeue.t`